extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -250,6 +256,14 @@ export class AmpAudio extends AMP.BaseElement {
       pauseHandler
     );
   }
+
+  /**
+   * @param {string} eventType
+   * @private
+   */
+  analyticsEvent_(eventType) {
+    triggerAnalyticsEvent(this.element, eventType);
```

Comment:
nit: Can we inline the `triggerAnalyticsEvent` function instead of creating a private method for it? It seems not worth it to create a private method for one line of code. Thanks.